change/@ni-nimble-components-c52231dc-0a44-4eed-8cba-f64e61b0ae15.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Placeholder comment to block completion. These styling changes will sort of mess up the chat appearance in SLE so I want to wait to merge this until I can block the Renovate update and submit the PR to fix that, probably next week.

[Pull Request 1003566](https://dev.azure.com/ni/DevCentral/_git/Skyline/pullrequest/1003566): Remove Nigel styling that Spright now provides